compiler-cli/src/lib.rs

```diff
@@ -167,9 +167,11 @@ enum Command {
     ///
     /// This command uses this environment variables:
     ///
-    /// - HEXPM_USER: (optional) The Hex username to authenticate with.
-    /// - HEXPM_PASS: (optional) The Hex password to authenticate with.
+    /// - HEXPM_USER: (optional, deprecated) The Hex username to authenticate with.
+    /// - HEXPM_PASS: (optional, deprecated) The Hex password to authenticate with.
```

Comment:
Could you remove the documentation for the deprecated environment variable please 🙏  We don't want to tell people to use things that are deprecated.

Reply:
absolutely!! both variables removed!